packages/private-design-output/src/example.html

```diff
@@ -357,399 +414,555 @@ <h2>Accessible accordions</h2>
     </div>
     <br/>
 
-    <!-- card -->
+    <!-- Card -->
     <div class="uq-grid">
-      <div class="uq-grid__col uq-grid__col--6">
+      <div class="uq-grid__col uq-grid__col--6"> 
+
         <h2>Cards</h2>
+        <!-- Default card -->
+        <h3>Default card</h3>
+
+        <div class="uq-grid">
+          <div class="uq-grid__col uq-grid__col--6 uq-grid__col--md-4 uq-grid__col--xl-3">
+            <div class="uq-card">
+              <div class="uq-card__image__wrapper">
```

Comment:
Some discrepancies with classnames here and card component, e.g. uq-card__image-wrapper, uq-card__content-wrapper, uq-card__title-super, might need to check through all the cards.